geologist/child-theme.json

```diff
@@ -317,14 +317,14 @@
 				"elements": {
 					"h2": {
 						"typography": {
-							"fontSize": "min(max(28px, 5vw), 38px)"
+							"fontSize": "var(--wp--preset--font-size--extra-large)"
 						}
 					}
 				}
 			},
 			"core/query-pagination": {
 				"typography": {
-					"fontSize": "18px",
+					"fontSize": "var(--wp--preset--font-size--normal)",
```

Comment:
Should this one be small?

Reply:
No, in Quadrat I first updated these to 20px then to the variable. Here I scaled it up correctly to the variable. See https://github.com/Automattic/themes/pull/4683#discussion_r714278294